### ISSUE TITLE
exercise 8.3: every other

### ISSUE THREAD

**Issue by jgmbenoit** [AllenDowney/ThinkPython2#96]
In exercise 8.3, we read

> A step size of 2 means every other character;(...).

As non native English reader, I get confuse: is `every other` an idiom  that means `every two` ?
I would favour plain English.

**Comment by AllenDowney** [comment]
Yes "every other character" means take one, skip one, take one, skip one, etc. But I understand that this idiom might not be familiar to non-native speakers.

An alternative is to say "every second character". Would that be easier to interpret? Or should I make it explicit, like "take one, skip one"?

Thanks!
Allen

**Comment by jgmbenoit** [comment]
"every second character" would be consistent with the spirit of the sentence, the (...) is in fact ``3 means every third, etc.''
(BTW, not sure of the semi-colon).
Best wishes,
Jerome

**Comment by OogieM** [comment]
Every second character is much harder for me to figure out. More explicit take one skipone seems more universal.

**Comment by jgmbenoit** [comment]
I agree with @OogieM to say that `` take one, skip one'' is better for a wider audience. A long this line, I suggest:

> A step size of 2 means take one character, skip one character; 3 means take one character, skip 2 characters; etc.
